src/main/java/com/mkdk72ki/asgmt10/controller/UserController.java

```diff
@@ -29,4 +36,12 @@ public ResponseEntity<User> findById(@PathVariable int id){
         User user = userService.findById(id);
         return ResponseEntity.ok(user);
     }
+
+    @PostMapping("/user")

```

Comment:
RESTfulにつくるなら通常扱うリソースは複数形で扱いますね。
```suggestion
    @PostMapping("/users")

```

Reply:
9543200e1662d0e8f0962ac41e1cbef514cec124 対応しました。ご確認お願いいたします。